### PR TITLE
unify:0.6.0

### PR DESCRIPTION
**Pull request by ChHecker**
I am submitting
- [ ] a new package
- [x] an update for a package

Changes:
- Multiple language support (by @SergeyGorchakov)
- Monetary units (by @benjamin-voisin)
- New units and small fixes

I have read and followed the submission guidelines and, in particular, I
- [x] selected a name that isn't the most obvious or canonical name for what the package does
- [x] added a `typst.toml` file with all required keys
- [x] added a `README.md` with documentation for my package
- [x] have chosen a license and added a `LICENSE` file or linked one in my `README.md`
- [x] tested my package locally on my system and it worked
- [x] `exclude`d PDFs or README images, if any, but not the LICENSE

### CODE REVIEW

File: packages/preview/unify/0.6.0/lib.typ [hunk 1]
```diff
@@ -0,0 +1,744 @@
+#let re-num = regex("^(-?\d+(\.|,)?\d*)?(((\+(\d+(\.|,)?\d*)-(\d+(\.|,)?\d*)))|((((\+-)|(-\+))(\d+(\.|,)?\d*))))?(e([-\+]?\d+))?$")
+#let unicode_exponents = (("\u2070", "0"), ("\u00B9", "1"), ("\u00B2", "2"), ("\u00B3", "3"), ("\u2074", "4"), ("\u2075", "5"), ("\u2076", "6"), ("\u2077", "7"), ("\u2078", "8"), ("\u2079", "9"), ("\u207A", "+"), ("\u207B", "-"))
```
Comment (elegaanz) [inline packages/preview/unify/0.6.0/lib.typ:2]: I'm not sure if this variable is intended for public use or not, but if not it should probably be indicated by adding a `_` in front as you did with other variables and functions. And if it is, it should probably use kebab-case as the rest of the package.

File: packages/preview/unify/0.6.0/lib.typ [hunk 2]
```diff
@@ -0,0 +1,744 @@
...
+  let data = lang-db.get()
+  if lang in data {
+    data.at(lang).prefixes
+  } else {
+    data.en.prefixes
+  }
+}
+
+#let unicode_exponent_list = for (unicode, ascii) in unicode_exponents {(unicode,)}
+#let exponent_pattern = regex("[" + unicode_exponent_list.join("|") + "]+")
```
Comment (elegaanz) [inline packages/preview/unify/0.6.0/lib.typ:240]: Same remark for these two variables.

**Review by elegaanz — COMMENTED** [review]
Just a comment about naming conventions, but it is not extremely important, especially since it was not noticed when reviewing previous versions, and changing these names may break people's documents. Otherwise everything looks good. So feel free to tell me if you just want to ignore my remark and merge as is.

**Comment by reknih** [comment]
@ChHecker is this still on your radar? :)

**Comment by ChHecker** [comment]
Sorry for the delay, I agree with your points and fixed them.

**Comment by elegaanz** [comment]
Thanks!

**Pull request merged**